### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by cenkkg**

### CODE REVIEW

**Review by abilalakin — APPROVED** [review]

File: .idea/misc.xml
Comment (DorukGercel) [inline .idea/misc.xml:1]: Remove the file pls :)

File: .idea/libraries/lib.xml
Comment (DorukGercel) [inline .idea/libraries/lib.xml:1]: Remove the file pls :)

File: .idea/libraries/junit_4_13_2_javadoc.xml
Comment (DorukGercel) [inline .idea/libraries/junit_4_13_2_javadoc.xml:1]: Remove the file pls :)

File: Connected Mobility Basics.iml
Comment (DorukGercel) [inline Connected Mobility Basics.iml:1]: Remove the file pls :)

**Review by DorukGercel — CHANGES_REQUESTED** [review]
Remove the unnecessary files pls, thanks for your work :)

**Pull request merged**